functime/cross_validation.py

```diff
@@ -18,12 +27,37 @@ def train_test_split(
 
     Returns
     -------
-    splitter : Callable[pl.LazyFrame, Tuple[pl.LazyFrame, pl.LazyFrame]]
-        Function that takes a panel LazyFrame and returns tuple of train / test LazyFrames.
+    splitter : Callable[pl.LazyFrame | pl.DataFrame, Tuple[pl.LazyFrame, pl.LazyFrame]]
```

Comment:
Shouldn't this coincide with above annotation? Namely
 ```suggestion
     splitter : Callable[[pl.LazyFrame | pl.DataFrame], Tuple[pl.LazyFrame, pl.LazyFrame] | Tuple[pl.DataFrame, pl.DataFrame]]
```

Reply:
thanks for spotting that!